app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditFragmentPresenter.kt

```diff
@@ -115,7 +115,11 @@ class ProfileEditFragmentPresenter @Inject constructor(
     dialogFragment.showNow(fragment.childFragmentManager, TAG_PROFILE_DELETION_DIALOG)
   }
 
-  /** Requests the specific profile to be deleted from the app and then routes the user either to administrator controls activity or profile list activity depending on whether they are currently using a tablet device. */
+  /** Requests the specific profile to be deleted from the app
+   * and then routes the user either to administrator controls
+   * activity or profile list activity depending on whether
+   * they are currently using a tablet device.
+   */
```

Comment:
When writing multi-line KDocs, they should be wrapped as so:

```kotlin
/**
 * Summary fragment...
 */
```

Also, you should have each line go up to the 100 character line limit before wrapping unless you're starting a new paragraph (in which case there should be a blank document line between the two paragraphs).

Reply:
done